007_smoothed_Laplace/smoothed_laplace.ipynb

```diff
@@ -0,0 +1,598 @@
hITt3rRqJIIr8IrW5KbhiCIQiAxUsFs64zgM7f/BXf88lcl6VpJlAe28NAUER6FX1Li1CyPIIgCIDFSwaTW3oOXtOvwze6vAKsXAOvvK/WQiEmI7ZLRZAmaTJYRgiAKh8rBVyrhPZjz6n9CEKwsCGYCj67kPRtK1KuBmJzYwaqaLEKzLCMUM0KMF3EdGE/tK4uAh2bOUYd2aaXSvQUCBpzBzODNo0iMEAXgdtNosiVGKJuGGAfiOvDybqBvHMPeghpw4lQSJKMNuWkqldrZYAN/fkHiXSwJogDITUOUCt3kQkSVgKA69jdV4p9XyOHd19eHlStXYvr06fB6vTjxxBOxdu3ajG2uuOIKCIKQcTvhhBOGfN81a9Zg7ty5CIVCuPzyy5FMJp3nwuEw5s6di507d+Y1xmeeeQYf+chHUFdXB5/Ph8MOOwz/8R//gT17eCzhs88+C0EQ0NPTk/8XLxASI5VKqBX/OPwb0Bk/BJggAeevJqsIUTAZbhqZ3DTE+KNJgFcZ+5smFT62a665BmvWrMGvf/1rvPXWW1i6dCnOOussZ6K3Oeecc9De3u7cHnvssZzvaZom/vVf/xXLly/Hyy+/jNdeew0/+9nPnOe/8pWvYPny5Zg2bdqw4/vJT36Cs846C83NzfjDH/6AjRs34u6770Y4HMYtt9xS+BcuEjI0VTBvN30C179ejxnifvxs5UUINk4v9ZCISUjcsYy4Y0bIMkIQsVgMf/jDH/DnP/8Zp5xyCgDgpptuwp/+9Cfcdddd+M53vuNsq2kampub83rfzs5OHDhwACtWrIDH48HHPvYxbNy4EQDw0ksvYd26dbjjjjuGfZ/du3fjuuuuw3XXXYf//d//dR6fMWMGTjnllDG1hAyELCMVTDxlYB/q8A/zMMS8TaUeDjFJcSwjStpNQ6m9BAHoug7DMAZ1svV6vXjxxRczHnv22WfR2NiIuXPn4tprr0VHR0fO921oaEBLSwv+9re/IRaL4YUXXsDChQuRTCbx2c9+FnfffTckaXgzzsMPP4xkMokvf/nLWZ+vrq4e/kuOEiRGKpiYa8KIJ2klSxSHbQXxyOk6IxTAShBAMBjEkiVLcPPNN2Pv3r0wDAO/+c1v8Oqrr6K9vd3Z7txzz8X999+Pp59+GrfccgvWrl2LM844A4lE9shcQRDw0EMP4eabb8Zhhx2Go48+GldddRW+973v4cwzz4TX68VJJ52EefPm4fbbb885vvfffx9VVVVoaWkZ9e9eKOSmqWDirgkjTj5+okicAFaFAlgJYiC//vWvcdVVV6G1tRWSJGHRokW49NJLsX79emebiy++2Pl7wYIFOOaYYzB9+nT83//9Hz75yU9mfd8PfehDGYGwmzdvxq9//Wts2LABp5xyClauXIlzzjkHCxYswCmnnIKFCxcOeg/GGARBGMVvWzxkGalg3AKEzOpEsTipvRTAShCDmD17Np577jn09/dj165deO2115BKpTBz5sycr2lpacH06dPx/vvv5/UZjDF85jOfwS233ALTNLFhwwZ86lOfQmNjI0499VQ899xzWV83d+5chMPhDCtNqSAxUsHEk+kJI5akyYMojuzZNGQZIQg3fr8fLS0tOHjwIJ588kl8/OMfz7ltV1cXdu3albf75Be/+AXq6urwsY99DIbBr+WpVMq5tx8byKc+9Smoqorvf//7WZ8fzwBWctNUMBmWEZo8iCLJqDNiN8qjmBFiHEmM01qqmM958sknwRjDvHnz8MEHH+BLX/oS5s2bhyuvvBIA0N/fj5tuugkXXnghWlpasH37dnzta19DfX09LrjggmHfv6OjA9/5znfw0ksvAQBqamowf/58rF69GkuXLsVTTz2Fr33ta1lf29bWhv/93//F5z//efT29mLZsmWYMWMGdu/ejfvuuw+BQGDc0ntJjFQwGTEj5KYhiiRrBVZy0xDjgCzyiqh9CWC8jLtBjX9uvoTDYdx4443YvXs3amtrceGFF+K///u/oSgKAECSJLz11lu477770NPTg5aWFpx++ul48MEHEQwGh33/L3zhC/jiF7+I1tZ0jah77rkHl19+OX70ox/hS1/6Eo477ricr1+xYgXmzp2LH/7wh7jgggsQi8UwY8YMfPSjH8UNN9yQ/xcdISRGKhi3ACExQhSLfeyQm4YYbzwyL80+kXvTXHTRRbjoootyPu/1evHkk08WPZ4HHnhg0GPHHXccNm3alPd7nHXWWTjrrLNyPn/aaaeBMVbU+PKFxEgFEyMxQowC6QBWqjNCjD/UI6Y8oADWCibTTUMrWaI4nDojClVgJQiiOEiMVDAJsowQo0AiRY3yCIIYGSRGKhi3myZGYoQokow6IwoFsBIEUTgkRiqYzABWWskSxZGuwOoKYKXjiSCIAiAxUsFQai8xGmQLYCU3DUEQhUBipEJhjFE5eGJUiGetwErHE0EQ+UNipEJJ6CbcaeMkRohiSVdgdXXtJcsIQRAFQGKkQhno06eYEaJYnN40CtUZIQiiOEiMVCjxAWb0gf8TRL7kqjMy1hUbCYIoH0iMVCgDu/RS116iWDIa5VmWEcaAlEFihCCuuOIKCIKA5cuXD3puxYoVEAQBV1xxxfgPbIJBYqRCGWwZITcNURwZdUZcHcQoiJUgOG1tbfjd736HWCzmPBaPx/HAAw9g2rRpJRzZxIHESIUyMEYkQT5+oggYY0jmFCMkcImxgzGGaCpakluhLshFixZh2rRpeOSRR5zHHnnkEbS1teHoo4/O+E7f//73MWvWLHi9Xhx55JH4/e9/7zxvGAauvvpqzJw5E16vF/PmzcNtt92W8VlXXHEFPvGJT+CHP/whWlpaUFdXh8997nNIpVJF7unxgVoMVSgDAwypAitRDG7BoSkSBEGAKotI6iaJEWJMiekxHP/b40vy2a9e+ip8iq+g11x55ZX41a9+hX/9138FAPzyl7/EVVddhWeffdbZ5j//8z/xyCOP4K677sKcOXPw/
```

Comment:
Explain here that the signal we use has many zeros and is piecewise constant with a single contiguous block of 1-values in the middle. Move the statement of what we assume about x to just before the cell where we set the Laplace prior. Discuss briefly if this is a sensible choice of prior for this signal (ie it will likely help get many zeros in the reconstruction but does not enforce it to be piecewise constant / nonzero values to be contiguous.

Reply:
Done